src/test/cypress/support/constants.ts

```diff
@@ -1,3 +1,17 @@
 // Constants which are used in the test specs
 
+// Requests
+export const DELETE = 'DELETE';
+export const POST = 'POST';
+export const GET = 'GET';
+export const PUT = 'PUT';
+export const BASE_API = 'api/';
+
+// Authentication
 export const authTokenKey = 'jhi-authenticationtoken';
+
+// Assertions
+export const beVisible = 'be.visible';
```

Comment:
Is this really necessary?

Reply:
Not necessary, but it is a performance improvement since this string is used quite frequently. I agree though that it might not be the best idea to extract it into the constants file because we definetely also have other strings which are often reused and might not be shared in this way.